x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/string/LocateFunctionPipe.java

```diff
@@ -82,7 +82,7 @@ public LocateFunctionProcessor asProcessor() {
         return new LocateFunctionProcessor(pattern.asProcessor(), source.asProcessor(), start == null ? null : start.asProcessor());
     }
     
-    public Pipe source() {
+    public Pipe sourcePipe() {
```

Comment:
Same comment here as the one for `Insert` function: maybe change `source` to `sourceText` and don't use `Pipe` in the return method name.